src/current/cockroachcloud/advanced-cluster-management.md

```diff
@@ -98,18 +98,18 @@ You can add up to nine regions at a time through the Console. See [Planning your
 
 1. On the **Regions** page, click **Add a region**.
 
-    If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions.
+    If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions. For GCP, each region requires a `/19` CIDR block.
```

Comment:
Same comment as below, making it clear that we configure the `/19` CIDR block out of the initially configured CIDR block

Reply:
Done, PTAL